casr/tests/casr_tests/arm_bin/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Did you test casr-core on AARCH64?

Reply:
offline mode works fine according to tests.rs and CI